openpype/hosts/flame/hooks/pre_flame_setup.py

```diff
@@ -33,15 +36,24 @@ def execute(self):
 
         """Hook entry method."""
         project_doc = self.data["project_doc"]
+        project_name = project_doc["name"]
+
+        # get image io
+        project_anatomy = get_anatomy_settings(project_name)
+        imageio_flame = project_anatomy["imageio"]["flame"]
```

Comment:
You've added these settings in this PR but they are not automatically added to a all projects. I would recommend to do it safely and use defaults for cases that values are not avalable in anatomy or are set to empty strings.

If the values are required then it would be good to raise `ApplicationLaunchFailed` when values are missing.